build-utils/css/utils/__tests__/isMatchedLastThemeSnapshot.test.ts

```diff
@@ -0,0 +1,123 @@
+import { readFileSync } from 'fs';
+
+import { darkStyleList } from '../../constants/_darkTheme';
+import { theme as lightTheme } from '../../constants/theme';
+import { isMatchedLastThemeSnapshot } from '../isMatchedLastThemeSnapshot';
+
+// const lightThemeCSSSnapshot = readFileSync(
+//   `build-utils/css/utils/__snapshots__/theme.css`,
+//   {
+//     encoding: 'utf-8',
+//   }
+// );
```

Comment:
```suggestion

```

Reply:
Is there a reason this was resolved, it looks like the comment is still there, but I might have missed a conversation.

Reply:
Hmn, it's weird. I removed it already 🤔 Have you reloaded your browser? 🤔

Reply:
Weird. I had to close and reopen the browser 🤔

Reply:
Powerful cache from git 😆